rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp

```diff
@@ -783,7 +783,7 @@ void FleetUpdateHandle::Implementation::fleet_state_publish_period(
   if (value.has_value())
   {
     fleet_state_timer = node->create_wall_timer(
-      std::chrono::seconds(1), [this]() { this->publish_fleet_state(); });
```

Comment:
This function block `fleet_state_publish_period()` can be removed completely. since it is not used.

Reply:
Got it, done!